src/plugins/tabs/components/tabs.vue

```diff
@@ -143,6 +128,17 @@ export default {
                 this.setCurrentTab = false;
             }
         },
+        removeItem (identifier) {
+            let pos = _.findIndex(this.tabsList, (tab) => 
+                this.openmct.objects.makeKeyString(tab.model.identifier) === this.openmct.objects.makeKeyString(identifier)
```

Comment:
Wouldn't using makeKey string avoid causing issues in the future if we were to change the shape/form of identifiers? But i changed it to your suggestion

Reply:
Yeah, that's a fair point. Identifier objects are how we have chosen to unique identify objects, so I think we should use them for that purpose. That said, my suggestion is quite verbose. This might be a case where we need some additional api. eg.
```
openmct.objects.areEqual(identifier1, identifier2);
```